ddtrace/llmobs/_llmobs.py

```diff
@@ -71,6 +71,9 @@ def enable(cls, tracer=None):
         if not config._dd_api_key:
             cls.enabled = False
             raise ValueError("DD_API_KEY is required for sending LLMObs data")
+        if not config._llmobs_ml_app_name:
+            cls.enabled = False
+            raise ValueError("DD_LLMOBS_APP_NAME is required for sending LLMObs data")
```

Comment:
Can we make this a little more actionable? Ex:
```suggestion
            raise ValueError("DD_LLMOBS_APP_NAME is required for sending LLMObs data. Ensure this configuration is set before running your application.")
```